Makefile

```diff
@@ -184,3 +184,22 @@ verify-mdlint:
 ### verify-all:           Verify all verify- rules.
 .PHONY: verify-all
 verify-all: verify-codegen verify-license verify-mdlint
+
+### update-codegen:       Update the generated codes (clientset, informer, deepcopy, etc).
+.PHONY: update-codegen
```

Comment:
We may need to remove L165 `codegen` and check if it is being used elsewhere (docs, etc).

Reply:
Done.